recipes/opencv/4.x/conanfile.py

```diff
@@ -219,7 +220,8 @@ def requirements(self):
         if self.options.get_safe("with_gtk"):
             self.requires("gtk/system")
         if self.options.dnn:
-            self.requires(f"protobuf/{self._protobuf_version}")
+            # Symbols are exposed https://github.com/conan-io/conan-center-index/pull/16678#issuecomment-1507811867
+            self.requires(f"protobuf/{self._protobuf_version}", transitive_libs=True)
```

Comment:
it's not true, it's likely a bug of conan client.

Reply:
Is it possible to change this comment? It's wrong. It's a workaround and it should be removed when fixed in conan client.